test/Conversion/TorchOnnxToTorch/simple_ops_q_to_z.mlir

```diff
@@ -517,6 +517,42 @@ func.func @test_selu(%arg0: !torch.vtensor<[3,4,5],f32>) -> !torch.vtensor<[3,4,
 
 // -----
 
+// CHECK-LABEL: func.func @test_reduce_max_default_axes_keepdim_example
+func.func @test_reduce_max_default_axes_keepdim_example(%arg0: !torch.vtensor<[3,2,2],f32>) -> !torch.vtensor<[1,1,1],f32> attributes {torch.onnx_meta.ir_version = 8 : si64, torch.onnx_meta.opset_version = 18 : si64, torch.onnx_meta.producer_name = "backend-test", torch.onnx_meta.producer_version = ""} {
+    // CHECK: torch.constant.int 0
+    // CHECK: torch.constant.int 1
+    // CHECK: torch.constant.int 2
+    // CHECK: torch.prim.ListConstruct %int0, %int1, %int2 : (!torch.int, !torch.int, !torch.int) -> !torch.list<int>
+    // CHECK: torch.constant.int 1
+    // CHECK: torch.aten.Bool.int %int1_0 : !torch.int -> !torch.bool
+    // CHECK: torch.aten.amax %arg0, %0, %1 : !torch.vtensor<[3,2,2],f32>, !torch.list<int>, !torch.bool -> !torch.vtensor<[1,1,1],f32>
+    %0 = torch.operator "onnx.ReduceMax"(%arg0) {torch.onnx.keepdims = 1 : si64} : (!torch.vtensor<[3,2,2],f32>) -> !torch.vtensor<[1,1,1],f32>
+    return %0 : !torch.vtensor<[1,1,1],f32>
+  }
+
+// -----
+
+// CHECK-LABEL: func.func @test_reduce_max_do_not_keepdims_example
+  func.func @test_reduce_max_do_not_keepdims_example(%arg0: !torch.vtensor<[3,2,2],f32>, %arg1: !torch.vtensor<[1],si64>) -> !torch.vtensor<[3,2],f32> attributes {torch.onnx_meta.ir_version = 8 : si64, torch.onnx_meta.opset_version = 18 : si64, torch.onnx_meta.producer_name = "backend-test", torch.onnx_meta.producer_version = ""} {
```

Comment:
What's unique about `Onnx::ReduceMax` is that it can run a reduce operation on mutliple axes. Whereas for `Torch::max` runs the reduce operation on just one axes.

Can you construct a test where you test this on multiple axes?

Reply:
NIT, I think `arg1` should be constructed from a literal tensor (or a tensor created from a function like `arange`). It would be easier to understand the test.